src/app/components/annotations/pages/verification/verification.component.ts

```diff
@@ -146,46 +172,8 @@ class VerificationComponent
     this.doneInitialScroll = true;
   }
 
-  protected handleDecision(decisionEvent: Event): void {
-    if (!this.isDecisionEvent(decisionEvent)) {
-      console.error("Received invalid decision event", decisionEvent);
-      return;
-    }
-
-    this.hasUnsavedChanges = true;
-
-    const subjectWrappers = decisionEvent.detail;
-    for (const subjectWrapper of subjectWrappers) {
-      const subject = subjectWrapper.subject as Readonly<AudioEvent>;
-
-      // I have to use "as string" here because the upstream typing is incorrect
-      // TODO: We should remove this "as string" and improve the upstream typing
-      const mappedDecision =
-        confirmedMapping[subjectWrapper.verification.confirmed as string];
-
-      const tagId = subjectWrapper.tag.id;
-
-      const verificationData: IVerification = {
-        audioEventId: subject.id,
-        confirmed: mappedDecision,
-        tagId,
-      };
-
-      const verification = new Verification(verificationData, this.injector);
-
-      // I have to use "as any" here to remove the readonly typing
-      const apiRequest = this.verificationApi.createOrUpdate(
-        verification,
-        subject as AudioEvent,
-        this.session.currentUser
-      );
-
-      // I use firstValueFrom so that the observable is evaluated
-      // but I don't have to subscribe or unsubscribe.
-      // Additionally, notice that the function is not awaited so that the
-      // render thread can continue to run while the request is being made
-      firstValueFrom(apiRequest);
-    }
+  protected tagTextFormatter(tag: Tag): string {
+    return tag.text;
```

Comment:
Can tag ever be undefined? Do we need a safe access operator?

Reply:
I think hard fail is the wrong choice because this will be really frustrating. 

But if we're confident it will never happen, that's fine with me